### PR TITLE
Feature/atr 113

### PR DESCRIPTION
**Pull request by metallmonster**

### CODE REVIEW

File: src/Acuminator/Acuminator.Analyzers/StaticAnalysis/ActionHandlerAttributes/ActionHandlerAttributesFix.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            var codeAction = CodeAction.Create(
+                codeActionName,
+                cancellation => FixActionHandlerAttributes(context.Document, context.Span, option, cancellation));
+
+            context.RegisterCodeFix(codeAction, context.Diagnostics);
+        }
+
+        private async Task<Document> FixActionHandlerAttributes(Document document, TextSpan span, FixOption option, CancellationToken cancellation)
+        {
+            if (cancellation.IsCancellationRequested)
```
Comment (SENya1990) [inline src/Acuminator/Acuminator.Analyzers/StaticAnalysis/ActionHandlerAttributes/ActionHandlerAttributesFix.cs]: @metallmonster shouldn't we use ThrowIfCancellationRequested?

Comment (metallmonster) [reply src/Acuminator/Acuminator.Analyzers/StaticAnalysis/ActionHandlerAttributes/ActionHandlerAttributesFix.cs]: ok

**Comment by metallmonster** [comment]
@SENya1990 one more try

**Comment by metallmonster** [comment]
Merge conflicts have been resolved

**Pull request merged**